src/Logging/index.js

```diff
@@ -0,0 +1,29 @@
+function logInfo(message) {
```

Comment:
I'm wondering if `src/Logging/index.js` is the right place for this file. It might make more sense to have as another file in our `services` directory instead, e.g. `src/data/services/NewRelicService.js`. Thoughts?